CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## Current Develop Branch
 
 ## 9.3.0 Fri Mar 26 2021
+- [#10721](https://github.com/MetaMask/metamask-extension/pull/10721): Swaps support for the binance network
+- [#10658](https://github.com/MetaMask/metamask-extension/pull/10658): Swaps support for forked mainnet on localhost
+- [#10650](https://github.com/MetaMask/metamask-extension/pull/10650): Fix: ETH 'token' now only appears once in the swaps to and from dropdowns.
```

Comment:
Nit: It's probably clearer to leave out the "token" part. Also quoting "to" and "from" makes this a bit easier to read, as it makes clear that it's referring to fields and not part of the sentence.
```suggestion
- [#10650](https://github.com/MetaMask/metamask-extension/pull/10650): Fix: ETH now only appears once in the swaps "to" and "from" dropdowns.
```